core/eval.go

```diff
@@ -97,6 +102,9 @@ func evalGET(args []string) []byte {
 	return Encode(obj.Value, false)
 }
 
+// Returns the Time-to-Live for the specified key if it exists
+// Returns -1 in case no expiration is set on the key
+// Returns -2 in case key doesn't exist
```

Comment:
Every command needs to have a standard verbose comment. The format would help us standardize code commenting across the repository. The comment we add in this file will lay the foundation.